src/main/scala/com/lambdalab/thriftdao/MongoThriftDao.scala

```diff
@@ -143,11 +154,18 @@ trait MongoThriftDao[T <: ThriftStruct, C <: ThriftStructCodec[T]] extends DBObj
   }
 
   def find(condition: (TField, Any)*): Iterator[T] = {
-    select(condition.map(convertAssoc) : _*).find()
+    select(condition.map(convertAssoc): _*).find()
+  }
+
+  def findWithBatch(field: TField, keys: List[Any], condition: (TField, Any)*): Iterator[T] = {
```

Comment:
keys means primary key?

Reply:
No, they are the values of `field`
You can see the examples in the test.